agent/src/policy/policy.rs

```diff
@@ -353,7 +353,36 @@ impl Policy {
         endpoints.dst_info.l3_epc_id
     }
 
+    // NOTE: This function has insufficient performance and is only used in low PPS scenarios.
+    // Currently, only the Integration collector is calling this function.
     pub fn lookup_all_by_epc(
```

Comment:
// NOTE: This function has insufficient performance and is only used in low PPS scenarios.
    // Currently, only the Integration collector is calling this function.